src/test/java/api/handlers/CloseAgedToLostLoanWhenLostItemFeesAreClosedApiTests.java

```diff
@@ -0,0 +1,127 @@
+package api.handlers;
+
+import static api.support.matchers.ItemMatchers.isAgedToLost;
+import static api.support.matchers.ItemMatchers.isCheckedOut;
+import static api.support.matchers.ItemMatchers.isLostAndPaid;
+import static api.support.matchers.LoanMatchers.isClosed;
+import static api.support.matchers.LoanMatchers.isOpen;
+import static org.folio.circulation.support.JsonPropertyFetcher.getUUIDProperty;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.ItemBuilder;
+import io.vertx.core.json.JsonObject;
+import lombok.val;
+
+public class CloseAgedToLostLoanWhenLostItemFeesAreClosedApiTests extends APITests {
+  private IndividualResource loan;
+  private IndividualResource item;
+
+  @Before
+  public void createLoanAndAgeToLost() {
+    feeFineOwnerFixture.cd1Owner();
+    feeFineTypeFixture.lostItemFee();
+    feeFineTypeFixture.lostItemProcessingFee();
+
+    val result = ageToLostFixture.createLoanAgeToLostAndChargeFees(
+      lostItemFeePoliciesFixture.ageToLostAfterOneMinutePolicy()
+        .withName("Age to lost policy")
+        .withSetCost(10.0)
+        .chargeItemAgedToLostProcessingFee(15.00));
+
+    item = result.getItem();
+    loan = result.getLoan();
+  }
+
+  @Test
+  public void shouldCloseLoanWhenAllFeesClosed() {
+    feeFineAccountFixture.payLostItemFee(loan.getId());
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId());
+
+    eventSubscribersFixture.publishLoanRelatedFeeFineClosedEvent(loan.getId());
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), isClosed());
+    assertThat(itemsClient.getById(item.getId()).getJson(), isLostAndPaid());
+  }
+
+  @Test
+  public void shouldDisregardNonLostFeeTypes() {
```

Comment:
Does disregard mean close the loan even when this fee is still open (and don't alter the fee itself)?

Reply:
Yes, correct.

Reply:
I think ignore might be a better word than disregard, maybe `shouldIgnoreFeesThatAreNotDueToLosingItem`? 

What do you think (I found it hard to express the negative well)?

Reply:
Renamed